src/pages/Categories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h2 className="categories">
+      Under construction
+    </h2>
+  </div>
```

Comment:
- [ ] Good job implementing the categories page, but it should display a  `Check Status` button. Please kindly ensure it displays the right element

Reply:
Oh thanks!, I already fixed it